onnx/defs/reduction/defs.cc

```diff
@@ -134,8 +134,10 @@ std::function<void(OpSchema&)> ArgReduceDocGenerator(const char* name) {
   return [=](OpSchema& schema) {
     std::string doc = R"DOC(
 Computes the indices of the {name} elements of the input tensor's element along the 
-provided axis. The resulted tensor has the same rank as the input if keepdims equal 1.
+provided axis. The resulted tensor has the same rank as the input if keepdims equal 1. 
```

Comment:
nit: Consider replacing "resulted tensor" with "resulting tensor" or just "output tensor" everywhere in these two ops. I think that is grammatically more correct.